### ISSUE TITLE
Incorrect model name when loading multiple models

### ISSUE THREAD

**Issue by robinderat** [UKPLab/EasyNMT#19]
I ran into a bug where a model keeps getting loaded, even after being used right before.

I set max_loaded_models=1
I then did a call to translate a sentence from english to dutch, loading the EN-NL model
Then I did a translate call from german to dutch, unloading the EN-NL model and loading DE-NL.

However, when I repeat the german to dutch call, the model gets loaded again.

Printing the self.models of the translator show the following:
`{'Helsinki-NLP/opus-mt-EN-NL': {'tokenizer': PreTrainedTokenizer(name_or_path='Helsinki-NLP/opus-mt-DE-NL'`

In other words, It loads the german model under the english name.

I found the problem in the source code:

In [load_model](https://github.com/UKPLab/EasyNMT/blob/db9a17d56a4563903dd2bc325c0541a44b92325d/easynmt/models/OpusMT.py#L23) the line `for model_name in self.models:` overwrites the value of model_name, which should indicate the model you want to load. Therefore, it puts the new model under the name of the last evaluated model in self.models.

**Comment by nreimers** [comment]
Great, thank for spotting this. I fixed it and will be part of the next release (which I hope to release soon)
